tests/functional/z_cluster/test_multiple_mds.py

```
@@ -178,3 +185,211 @@ def test_node_drain_and_fault_tolerance_for_multiple_mds(self, pod_factory):
         assert err_count == 0, (
             f"IO error on pod {pod_obj.name}. " f"FIO result: {fio_result}"
         )
+
+
+EC_POOL_NAME = "mds-test-ec-fs"
+EC_DATA_CHUNKS = 2
+EC_CODING_CHUNKS = 2
+
+
+@pytest.fixture()
```

Ruff (0.15.21)

[warning] Use `@pytest.fixture` over `@pytest.fixture()`

Remove parentheses

(PT001)